en/docs/users/01_Installation.md

```diff
@@ -48,7 +48,49 @@ As its name suggests, it is the working release for developers. **This release i
 
 # Nginx installation
 
-**TODO**
+This is an example nginx configuration file. It covers http, https and php-fpm configuration.
+
+```
+server {
+  listen 80; # http on port 80
+  listen 443 ssl; # https on port 443
+
+  # https configuration 
+  ssl on;
+  ssl_certificate      /etc/nginx/server.crt;
+  ssl_certificate_key  /etc/nginx/server.key;
+
+  # your server's url(s)
+  server_name example.com rss.example.com;
+
+  # the folder p of your FreshRSS installation
+  root /srv/FreshRSS/p/;
+
+  index index.php index.html index.htm;
+
+  # nginx log files
+  access_log /var/log/nginx/rss.access.log;
+  error_log /var/log/nginx/rss.error.log;
+
+  # php files handling
+  # this regex is mandatory because of the API
+  location ~ ^/(.*)\.php(.*)$ {
```

Comment:
Isn't the configuration suggested on http://wiki.nginx.org/PHPFcgiExample#Connecting_nginx_to_PHP_FPM working?
In any case, I believe a `/` after `.php` would be desirable to avoid matching e.g. `.phps`, with e.g. `[^/]\.php(/|$)` or `^(.+?\.php)(/.*)?$`